docs/src/references/c8y-configuration-management.md

```diff
@@ -162,4 +162,235 @@ Note that:
   for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
 * Since the type of configuration file is used as an MQTT topic name, the characters `#` and `+` cannot be used in a type name.
   If such a character is used in a type name (or in the path of a configuration file without explicit type),
-  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
\ No newline at end of file
+  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
+* Notifications for child-devices and their external devices are handled differently, as configuration files are fetched/pushed from/to external devices via HTTP filetransfer. Thereby notifications are part of the filetransfer contract (details see section [Fetching/Pushing configuration file](#fetchingpushing-configuration-file-fromto-the-external-device) below).
+
+## Configuration files for child devices
+
+To manage configuration files for child-devices the `c8y_configuration_plugin` aspects as below are relevant:
+
+* Associating with cloud's child-device twin, i.E. managing supported configuration list of child devices 
+  * an external child device declares its supported config-types to thin-edge via MQTT
+  * thin-edge maintaines the supported config-types of each child-device a separate config-file
+  * the plugin associates the reported config-type list from the external to its cloud device twin
+  * these config-list files per child-device is also managed by Configuration Management feature from C8Y
+* Consuming/Providing files from/to thin-edge via network
+  * any configuration file exchange to/from the external child device is handled by an HTTP based filetransfer service provided by thin-edge
+
+## Managing Supported Configuration List of child devices
+
+To announce a child-devices configuration management capability child-device provisioning as described in [child-device reference documentation](child-devices.md#1-child-device-provisioning) is used.
+
+The `capability specific JSON object` for `c8y_configuration_plugin` contains all configurations the external device provides.
+Thereby each configuration appears with a `type` and with an optional field `path`.
```

Comment:
If no configuration file path is mentioned, what is the need for such an entry? Should Every entry be associated with some config file right? Or will there be a default file/path?

Reply:
That `path` is rest of an discard approach that was accidentally not removed. I removed it now.
Thanks for the hint.